extensions/notebook/package.json

```diff
@@ -419,12 +419,6 @@
           "when": "false"
         }
       ],
-      "notebooks/title": [
```

Comment:
Does this not work anymore then? If so we should remove this contribution point.

Reply:
It doesn't work anymore and I couldn't find in the documentation an alternative, that is different from using the view/title contribution point.

Reply:
It doesn't seem useful to keep it around so I'd say remove it. @chlafreniere What do you think?

Reply:
Do we know why it's broken? Did any of the other contribution points break as well?

Reply:
Yeah, something broke with our secondary menu contributions : https://github.com/microsoft/azuredatastudio/issues/15824

So it's pretty likely that fixing both at the same time would be relatively simple.

But in the end do we have a good reason for wanting it at the viewlet (container) level vs on the actual books view itself? I'm just not sure the feature is actually useful to have right now when there's already functionality built in that we could leverage.

Reply:
Also, I think it makes more sense to have the **create book command** and **add remote book command** on the same menu since both add the book to the book tree view at the end.

Reply:
Agreed there @barbaravaldez. Let's go with this.